demo/c/koala_demo_file.c

```diff
@@ -87,17 +88,21 @@ static void print_dl_error(const char *message) {
 }
 
 static struct option long_options[] = {
-        {"access_key",   required_argument, NULL, 'a'},
-        {"library_path", required_argument, NULL, 'l'},
-        {"model_path",   required_argument, NULL, 'm'},
-        {"input_path",   required_argument, NULL, 'i'},
-        {"output_path",  required_argument, NULL, 'o'},
+        {"access_key",              required_argument, NULL, 'a'},
+        {"library_path",            required_argument, NULL, 'l'},
+        {"model_path",              required_argument, NULL, 'm'},
+        {"device",                  required_argument, NULL, 'y'},
+        {"input_path",              required_argument, NULL, 'i'},
+        {"output_path",             required_argument, NULL, 'o'},
+        {"show_inference_devices",  required_argument, NULL, 'z'},
 };
 
 static void print_usage(const char *program_name) {
     fprintf(
             stdout,
-            "Usage: %s [-l LIBRARY_PATH -m MODEL_PATH -a ACCESS_KEY -i INPUT_PATH -o OUTPUT_PATH]\n",
+            "Usage: %s -l LIBRARY_PATH [-m MODEL_PATH -a ACCESS_KEY -y DEVICE -i INPUT_PATH -o OUTPUT_PATH]\n"
+            "        %s [-z, --show_inference_devices]\n",
```

Comment:
since the library is required for show inference devices
```suggestion
            "        %s [-z, --show_inference_devices] -l LIBRARY_PATH\n",
```